### PR TITLE
Feature/mx 1521 handle entity type from mex common

### PR DESCRIPTION
**Pull request by carolinscholl**
This PR solves MX-1521. 
- Although entityType attribute is now present in the models in mex-common, we temporarily still re-create the models from the base classes because of the static identifiers. The re-creation of models will be replaced once MX-1533 is solved.
- However, the entity types were renamed to keep the prefix "Extracted", to be consistent with the entityType values from mex-common.

### CODE REVIEW

File: mex/backend/merged/transform.py [hunk 1]
```diff
@@ -28,8 +29,14 @@ def transform_graph_results_to_merged_item_search_response_facade(
         try:
             model = transform_search_result_to_model(result)
             model_dict = model.model_dump()
-            model_dict["entityType"] = f"Merged{model_dict['entityType']}"
-            items.append(model_dict)
+            # create a MergedModel class with the dictionary
+            model_dict["entityType"] = model_dict["entityType"].replace(
+                "Extracted", "Merged"
+            )
+            del model_dict["hadPrimarySource"]
```
Comment (cutoffthetop) [inline mex/backend/merged/transform.py:36]: could just use `model_dump(exclude={"hadPrimarySource", "identifierInPrimarySource"})` above

**Review by cutoffthetop — CHANGES_REQUESTED** [review]
looks very good, just 2 small suggestions

File: mex/backend/merged/transform.py (hunk 1 above)
Comment (carolinscholl) [reply mex/backend/merged/transform.py]: True! Done in: 3224603dfdadce084d23cb81619c1c195ba4c682

File: mex/backend/ingest/models.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def get_all(self) -> list[AnyExtractedModel]:
         "BulkIngestRequest",
         __base__=_BaseBulkIngestRequest,
         **{
-            f"Extracted{name}": (list[model], [])
+            f"{name}": (list[model], [])
```
[the remark this answers is not in the document]
Comment (carolinscholl) [reply mex/backend/ingest/models.py]: Yes 🤦  Done here: eb3e7c29babd3ab7ec82d8337579de41f5bafc9c

**Review by cutoffthetop — APPROVED** [review]

**Review by cutoffthetop — APPROVED** [review]

**Pull request merged**